docroot/themes/contrib/civictheme/theme-settings.php

```diff
@@ -634,3 +635,388 @@ function _civictheme_path_field_description($original_path, $fallback_path) {
     '@local-file' => $local_file,
   ]);
 }
+
+/**
+ * Provide color settings to theme settings form.
+ *
+ * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+ * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+ */
+function _civictheme_form_system_theme_settings_colors(&$form, FormStateInterface &$form_state) {
+  $form['colors'] = [
+    '#type' => 'details',
+    '#title' => t('Colors'),
+    '#weight' => 40,
+    '#open' => TRUE,
+    '#tree' => TRUE,
+  ];
+
+  $form['colors']['use_brand_colors'] = [
+    '#title' => t('Use Brand colors'),
+    '#type' => 'checkbox',
+    '#default_value' => theme_get_setting("colors.use_brand_colors") ?? TRUE,
+  ];
+  // Brand.
+  $form['colors']['brand'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Brand colors'),
+    '#tree' => TRUE,
+    '#states' => [
+      'visible' => [
+        ':input[name="colors[use_brand_colors"]' => ['checked' => TRUE],
+      ],
+    ],
+  ];
+
+  // Brand light.
+  $form['colors']['brand']['light'] = [
+    '#type' => 'fieldset',
+    '#tree' => TRUE,
+    '#attributes' => [
+      'class' => 'color-light',
+    ],
+  ];
+  $form['colors']['brand']['light']['brand1'] = [
+    '#type' => 'color',
+    '#title' => t('Brand1'),
+    '#default_value' => theme_get_setting("colors.brand.light.brand1") ?? '#00698f',
+  ];
+  $form['colors']['brand']['light']['brand2'] = [
+    '#type' => 'color',
+    '#title' => t('Brand2'),
+    '#default_value' => theme_get_setting("colors.brand.light.brand2") ?? '#e6e9eb',
+  ];
+  $form['colors']['brand']['light']['brand3'] = [
+    '#type' => 'color',
+    '#title' => t('Brand3'),
+    '#default_value' => theme_get_setting("colors.brand.light.brand3") ?? '#121313',
+  ];
+
+  // Brand dark.
+  $form['colors']['brand']['dark'] = [
+    '#type' => 'fieldset',
+    '#tree' => TRUE,
+    '#attributes' => [
+      'class' => 'color-dark',
+    ],
+  ];
+  $form['colors']['brand']['dark']['brand1'] = [
+    '#type' => 'color',
+    '#title' => t('Brand1'),
+    '#default_value' => theme_get_setting("colors.brand.dark.brand1") ?? '#61daff',
```

Comment:
please replace default colors with `CIVICTHEME_BRAND_COLOR_LIGHT_BRAND1` constant (add the constant to the `constants.inc` file) for brand ALL colors

Reply:
Updated